src/styles/index.scss

```diff
@@ -1,3 +1,87 @@
-body {
+@import './variables';
+
+.page {
   margin: 0;
+  box-sizing: border-box;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: wrap;
+  flex-direction: row;
+
+  width: 800px;
+  min-width: 50vmin;
+  max-width: 100vmin;
+
+  padding: 10px;
```

Comment:
The `padding` property should be defined using a variable for consistency and maintainability, especially if the same padding value is used in multiple places.